src/Tooltip/tooltip.css

```diff
@@ -338,22 +497,174 @@
 {
     .tooltip
     {
-        background: var( --PC-RED );
+      background: var( --PC-RED );
+    }
+    &.position__top,
+    &.position__topLeft,
+    &.position__topRight
+    {
+      .tooltip
+      {
+        &::after
+        {
+          border-top-color: var( --PC-RED );
```

Comment:
Why set this individually for each position? I think you can just do `tooltip::after { border-color: ... }` one time and be done.... right? 🤔

Reply:
Done